File: Constructorio_Net/src/models/ResultFacet.java
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

/**
 * Constructor.io Item Facet
 */
*/
namespace Constructorio_NET
{
    public class ResultFacet {

        [JsonPropertyName("name")]
        public String name;

        [JsonPropertyName("values")]
        public List<String> values;

    }
}

File: src/Constructorio_NET/models/ResultFacet.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

/**
 * Constructor.io Item Facet
 */
namespace Constructorio_NET.Models
{
    public class ResultFacet {

        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("values")]
        public List<string> Values { get; set; }
    }
}